zookeeper-api/src/test/java/org/apache/helix/zookeeper/impl/client/TestRawZkClient.java

```diff
@@ -1063,23 +1063,25 @@ public void testAsyncWriteByExpectedSession() throws Exception {
    * Tests getChildren() when there are an excessive number of children and connection loss happens,
    * the operation should terminate and exit retry loop.
    */
-  @Test(timeOut = 30 * 1000L)
+  @Test
   public void testGetChildrenOnLargeNumChildren() throws Exception {
     final String methodName = TestHelper.getTestMethodName();
     System.out.println("Start test: " + methodName);
+    String[] nodePaths = new String[110001];
```

Comment:
I assume it's 110 * 1000 + 1 ? 
nit: Let's use constants and multiplication for these numbers. Same as below in the for-loop. Thanks!